features/admin/access_a_service.feature

```diff
@@ -30,6 +55,24 @@ Scenario: Admin with Service Manager role can edit, remove and publish a service
   Then I see a success banner message containing 'You published ‘Plant-based cloud hosting’.'
 
 
+@skip-preview
+Scenario Outline: Admins with Framework Manager and Support roles can view, but neither edit, nor remove services
+  Given I am logged in as the existing <role> user
+  And I am on the 'Admin' page
+  When I click '<link_name>'
+  Then I am on the '<link_name>' page
+  When I enter that service id in the 'service_id' field and click its associated 'Search' button
+  Then I am on that service's page
+  And I don't see the 'Edit' link
+  And I don't see the 'Remove service' link
+
+  Examples:
+    | role                    | link_name                                 |
+    | admin-framework-manager | View suppliers and services               |
+    | admin                   | Edit supplier accounts or view services   |
+
+
+@skip-staging @skip-production
```

Comment:
We won't need `skip-production` for this one as it's not a smoke test.

Reply:
Ah yes